All CI is broken during ip tool install for multus

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
All CI just started failing a couple hours ago during ip tool install for Multus:
```
Dockerfile:23
--------------------
  21 |     
  22 |     # install 'ip' tool for Multus
  23 | >>> RUN dnf install -y --setopt=install_weak_deps=False iproute && dnf clean all
  24 |     
  25 |     # Install the s5cmd package to interact with s3 gateway
--------------------
ERROR: failed to solve: process "/bin/sh -c dnf install -y --setopt=install_weak_deps=False iproute && dnf clean all" did not complete successfully: exit code: 1
make[2]: *** [Makefile:87: do.build] Error 1
make[1]: *** [Makefile:26: ceph.linux_amd64] Error 2
make: *** [Makefile:139: build] Error 2
```

Original issue for csv generation was not true, edited since this is not the root cause:
```
generate csv with latest operator-sdk
Generating bundle
Generating bundle manifests
Building a ClusterServiceVersion without an existing base
Bundle manifests generated successfully in ../../build/csv/ceph/amd64
Generating bundle metadata
Error: Process completed with exit code 1.
```

**Expected behavior:**
Passing CI

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->
See any of the recent CI such as [Canary tests](https://github.com/rook/rook/actions/workflows/canary-integration-suite.yml) or other [integration tests](https://github.com/rook/rook/actions/workflows/integration-tests-on-release.yaml). The build step is failing though, it's not related to the integration tests.

--- a/images/ceph/Dockerfile
+++ b/images/ceph/Dockerfile
@@ -20,7 +20,7 @@ ARG S5CMD_VERSION
 ARG S5CMD_ARCH
 
 # install 'ip' tool for Multus
-RUN dnf install -y --setopt=install_weak_deps=False iproute && dnf clean all
+RUN dnf install -y --repo baseos --setopt=install_weak_deps=False iproute && dnf clean all
 
 # Install the s5cmd package to interact with s3 gateway
 RUN curl --fail -sSL -o /s5cmd.tar.gz https://github.com/peak/s5cmd/releases/download/v${S5CMD_VERSION}/s5cmd_${S5CMD_VERSION}_${S5CMD_ARCH}.tar.gz && \

